lib/parse_packwerk/configuration.rb

```diff
@@ -37,7 +37,13 @@ def self.excludes(config_hash)
                    Array(specified_exclude)
                  end
 
-      excludes.push Bundler.bundle_path.join('**').to_s
+      begin
+        excludes.push Bundler.bundle_path.join('**').to_s
+      rescue Bundler::GemfileNotFound
+        # Not using a Gemfile, so we can't add the bundle path.
```

Comment:
Maybe we can expand a little more on the message to give future readers more context:

```suggestion
        # Optionally, add bundle to the path. Skip when there isn't a Gemfile.
```